### ISSUE TITLE
API using the HTTP protocol

### ISSUE THREAD

**Issue by tokyo2329** [tokyo2329/typer#1]
Current API uses the HTTP protocol, which is forbidden by most current browsers. (mismatching protocols when a site is hosted on HTTPS and it's getting resources from HTTP)

```
https://typeer.netlify.app/' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://metaphorpsum.com/sentences/1'. This request has been blocked; the content must be served over HTTPS.
```

We need to migrate to a new API that supports HTTPS.

**Comment by tokyo2329** [comment]
An idea;
https://random-word-api.herokuapp.com

Random words API, works great unfortunately the produced sentence won't have any logical (and grammatical) sense. Users would type on meaningless words

**Comment by tokyo2329** [comment]
Fixed with #d88cc75
